examples/simulator-1/index.rsh

```diff
@@ -1,5 +1,7 @@
 'reach 0.1';
 
+// rps-2-rps
```

Comment:
Consider changing the Makefile to do `cp ../rps-2-rps/index.rsh .` as part of the `build` process

Reply:
i'm not sure yet if i will change the original programs

Reply:
also, not doing that makes local testing more convenient